examples/catalog/bin/sample_page.dart

```diff
@@ -22,22 +22,29 @@ class SampleError extends Error {
 // beginning of lines within the comment. A keyword's value is all of
 // the following text up to the next keyword or the end of the comment,
 // sans leading and trailing whitespace.
-const String sampleCatalogKeywords = r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
+const String sampleCatalogKeywords =
+    r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
 
 Directory outputDirectory;
 Directory sampleDirectory;
 Directory testDirectory;
 Directory driverDirectory;
 
-void logMessage(String s) { print(s); }
-void logError(String s) { print(s); }
+void logMessage(String s) {
+  print(s);
+}
+
+void logError(String s) {
+  print(s);
+}
 
 File inputFile(String dir, String name) {
   return File(dir + Platform.pathSeparator + name);
 }
 
 File outputFile(String name, [Directory directory]) {
-  return File((directory ?? outputDirectory).path + Platform.pathSeparator + name);
+  return File(
+      (directory ?? outputDirectory).path + Platform.pathSeparator + name);
```

Comment:
yikes, definitely don't do anything like that.

if you really want this on multiple lines (why?) then the `)` should be on a line on its own. You can force dartfmt to do that with a trailing comma.